libavfilter/vf_dnn_detect.c

```diff
@@ -539,5 +539,5 @@ const AVFilter ff_vf_dnn_detect = {
     .inputs        = dnn_detect_inputs,
     .outputs       = dnn_detect_outputs,
     .priv_class    = &dnn_detect_class,
-    .activate      = dnn_detect_activate,
+    .activate      = dnn_detect_activate_async,
```

Comment:
still use dnn_detect_activate here.  It is not good to see 'async' or 'sync' here, since the filter will not care about this concept.  For less change, we may change the function name from dnn_detect_activate_async into dnn_detect_activate_impl, and call dnn_detect_activate_impl in function dnn_detect_activate.  and then delete function  dnn_detect_activate_sync.


static int dnn_detect_activate(AVFilterContext *filter_ctx)
{
    DnnDetectContext *ctx = filter_ctx->priv;

    if (ctx->dnnctx.async)  // do not remove this variable, but to add deprecated, refer to 'options' in dnn_filter_common.h
        output warning here to be backward compatible.
   
    return dnn_detect_activate_impl(filter_ctx);
}

the same comment for dnn_processing.

Reply:
ok, please ignore this comment, since I see it is fixed in the next patch.

Reply:
Sure.